src/shared/sam/cli/samCliBuild.ts

```diff
@@ -73,6 +76,7 @@ export class SamCliBuildInvocation {
             skipPullImage = false,
             ...params
         }: SamCliBuildInvocationArguments,
+        private readonly file: FileFunctions = getDefaultFileFunctions()
```

Comment:
Nit: For consistency, please wrap this in a context, i.e. `private readonly context: { file: FileFunctions } = ...`